src/Parts/MyTrainingCard/SingleTrainingCard.jsx

```diff
@@ -1,42 +1,57 @@
-import {
-  Image,
-  Card,
-  Col,
-  Row,
-  Button,
-  Space,
-  Typography,
-} from 'antd';
-import { EnvironmentOutlined, UserOutlined } from '@ant-design/icons';
-import { useNavigate } from 'react-router-dom';
-import './MyTrainingCard.css';
-import PropTypes from 'prop-types';
+import { Image, Card, Col, Row, Button, Space, Typography, Modal } from "antd";
+import { EnvironmentOutlined, UserOutlined } from "@ant-design/icons";
+import React, { useState } from "react";
+import { useNavigate } from "react-router-dom";
+import "./MyTrainingCard.css";
+import PropTypes from "prop-types";
 const { Text } = Typography;
+import { Rate } from "antd";
 
-export const SingleTrainingCard = (props) => {
+export const SingleTrainingCard = props => {
   const { item, id, location } = props;
   const navigate = useNavigate();
 
   const openLocation = () => {
-    console.log('test hit');
+    console.log("test hit");
```

Comment:
Please remove log!

Reply:
thanks for review, this issue already fix here 4a0144443efe14916dc188d008c22c4031fda99a

Reply:
https://www.loom.com/share/f4d62379539d4fa79f9116f2de1ec2e9